test/e2e/self_hosted.go

```diff
@@ -226,6 +226,21 @@ func SelfHostedSpec(ctx context.Context, inputGetter func() SelfHostedSpecInput)
 			Namespace:            namespace.Name,
 		})
 
+		log.Logf("Ensure clusterctl does not take ownership on any fields on the self-hosted cluster")
```

Comment:
Let's add a comment similar to 
note: we do expect that moves restores the managedFields existing in the old cluster, and thus removes any managedField entry with clusterctl as a manager; this should happen in all the objects processed by move, but for sake of simplicity we test only the Cluster object, which have also very specific processing for the paused field.